c2corg_ui/static/js/documentediting.js

```diff
@@ -552,25 +535,40 @@ app.DocumentEditingController.prototype.pushToArray = function(object, property,
   var pushed = app.utils.pushToArray(object, property, value);
   var checkbox = $(event.currentTarget).find('input');
 
-  if (pushed) {
-    checkbox.prop('checked', true);
+  checkbox.prop('checked', pushed);
+}
+
+/**
+ * @export
+ */
+app.DocumentEditingController.prototype.formatDate = function(date) {
```

Comment:
To move in ``app.utils``?
Date manipulating might be useful in the outings advanced search...